apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -182,18 +196,43 @@ export function StudentAssessmentInstance({
         </div>
 
         <div class="card-body">
-          ${!resLocals.assessment.allow_real_time_grading && resLocals.assessment_instance.open
-            ? html`
+          ${run(() => {
```

Comment:
nit: I typically like to break things out into components once they hit a complexity of 'we need a full function to render this'.

Reply:
Done in 7817250.